### DISCUSSION TITLE
QQ音源播放问题

### DISCUSSION THREAD

**Discussion by lyh20110** [1715173329/UnblockNeteaseMusic#579] [Q&A] [answered]
我下载最新的0.27exe文件，设置启用QQ音源，且系统变量也设置对应的变量，用周杰伦的歌曲做测试，发现有些歌曲无法播放，比如：我很忙专辑，都是加载失败，cmd输出如下：
DEBUG: (provider/match) Getting the audio...
    source: "qq"
    info: {
      "name": "鏈€闀跨殑鐢靛奖",
      "id": 185821,
      "alias": [],
      "duration": 235000,
      "album": {
        "name": "鎴戝緢蹇?,
        "id": 18886
      },
      "artists": [
        {
          "name": "鍛ㄦ澃浼?,
          "id": 6452
        }
      ],
      "keyword": "鏈€闀跨殑鐢靛奖 - 鍛ㄦ澃浼?
    }
DEBUG: (cache) Cleaning up the expired caches...
    cacheStorageId: "provider/qq"
DEBUG: (cache) Something did not hit. Storing the execution result...
    logKey: "Something"
    cacheStorageId: "provider/qq"
DEBUG: (request) Start requesting https://c.y.qq.com/soso/fcgi-bin/client_search_cp?ct=24&qqmusic_ver=1298&remoteplace=txt.yqq.center&t=0&aggr=1&cr=1&catZhida=1&lossless=0&flag_qc=0&p=1&n=20&w=%E6%9C%80%E9%95%BF%E7%9A%84%E7%94%B5%E5%BD%B1%20-%20%E5%91%A8%E6%9D%B0%E4%BC%A6&g_tk=5381&jsonpCallback=MusicJsonCallback10005317669353331&loginUin=0&hostUin=0&format=jsonp&inCharset=utf8&outCharset=utf-8&notice=0&platform=yqq&needNewCode=0
DEBUG: (request) Start requesting https://u.y.qq.com/cgi-bin/musicu.fcg?data=%7B%22req_0%22%3A%7B%22module%22%3A%22vkey.GetVkeyServer%22%2C%22method%22%3A%22CgiGetVkey%22%2C%22param%22%3A%7B%22guid%22%3A%226631492%22%2C%22loginflag%22%3A1%2C%22filename%22%3A%5B%22M800001y7CaR29k6YP.mp3%22%5D%2C%22songmid%22%3A%5B%22003ZrdgZ0UnjDl%22%5D%2C%22songtype%22%3A%5B0%5D%2C%22uin%22%3A%22529376719%22%2C%22platform%22%3A%2220%22%7D%7D%7D
DEBUG: (request) Start requesting https://u.y.qq.com/cgi-bin/musicu.fcg?data=%7B%22req_0%22%3A%7B%22module%22%3A%22vkey.GetVkeyServer%22%2C%22method%22%3A%22CgiGetVkey%22%2C%22param%22%3A%7B%22guid%22%3A%225710615%22%2C%22loginflag%22%3A1%2C%22filename%22%3A%5B%22M500001y7CaR29k6YP.mp3%22%5D%2C%22songmid%22%3A%5B%22003ZrdgZ0UnjDl%22%5D%2C%22songtype%22%3A%5B0%5D%2C%22uin%22%3A%22529376719%22%2C%22platform%22%3A%2220%22%7D%7D%7D
ERROR: (hook) All promises were rejected
    AggregateError: All promises were rejected

**Comment by 1715173329** [comment, marked as answer]
测试了一下是没问题的，请确认您已开通绿钻业务。

**Reply by lyh20110** [reply]
开通绿钻后重新配置一下，现在可以听了，谢谢，但发现有些歌曲匹配到其他版本或者伴奏，这个怎么弄？

**Reply by 1715173329** [reply]
由于有关标题问题已解决，故标记为 Answered。
匹配问题目前依赖于接口返回信息，暂时无解，参见 #428
